### PR TITLE
fix: top account navigation horizontal alignment

### LINKED ISSUE #3674: [Task]:  Account selection is off-center
### Task description

image

image

### Requirements

Center the account selection (top navigation) on mobile.

### Acceptance criteria

The requirement is completed.

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by paul-boegelsack**
## Summary

Deleted the x padding of the selection because it moved the selection too far to the left.

## Changelog

```
- Deleted x padding of the account selection
```

## Relevant Issues

Closes #3674

### Platforms

> Please select any platforms where your changes have been tested.

- __Desktop__
  - [ ] MacOS
  - [ ] Linux
  - [ ] Windows
- __Mobile__
  - [X] iOS
  - [X] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by begonaalvarezd — COMMENTED** [review]
IMO, the issue with the wrong alignment comes from the translate X calculation, not the paddings. 
I see couple of things we need to fix around this:
- Make sure the account top navigation is horizontally aligned.
- When you change the name of the account (and increase its name by a lot) the element stays in the previous position, this needs a new calculation

**Review by amadeu2 — CHANGES_REQUESTED** [review]
I've just tested and happens what Begoña says. Do you think it's doable to achieve?

**Comment by paul-boegelsack** [comment]
> IMO, the issue with the wrong alignment comes from the translate X calculation, not the paddings.

Your right, my fix attempt with the padding was brainless 😅 The calculation worked with the center of the button not with the center of the text.

> I've just tested and happens what Begoña says. Do you think it's doable to achieve?

 I tried 😄 Now the selection is aligned with its text and not with the button center itself. When the text resizes through a wallet name update it should recenter the selection again.

File: packages/shared/components/AccountNavigation.svelte [hunk 1]
```diff
@@ -34,41 +46,31 @@
...
+            <Text type="h4">{$selectedAccountStore?.alias}</Text>
+        </div>
         <div class="transform transition-transform {isDrawerOpened ? 'rotate-180' : 'rotate-0'}">
             <Icon icon="chevron-down" height="18" width="18" classes="text-gray-800 dark:text-white" />
         </div>
     </button>
     <Drawer bind:this={drawer} opened={isDrawerOpened} onClose={() => (isDrawerOpened = false)}>
-        <div class="flex flex-col p-5 pt-7 w-full safe-area safe-area">
-            {#if drawerRoute === 'create'}
+        fix/center-the-account-selection
```
Comment (begonaalvarezd) [inline packages/shared/components/AccountNavigation.svelte:69]: 🕵🏼 this looks like a wrong merge

Comment (paul-boegelsack) [reply packages/shared/components/AccountNavigation.svelte]: 😱 Thanks, you saved me.

**Pull request merged**